index.php

```diff
@@ -145,93 +145,94 @@
-                    <div class="dish-info">
-                        <div class="name">Piquant Teriyaki Shrimp Curry</div>
-                        <div class="price">$78</div>
-                        <button class="add-to-cart-button">Add to Cart</button>
-                    </div>
-                </div>
-                <div class="dish">
-                    <img src="Roasted Herb Vegetables Risotto.jpg" alt="Roasted Herb Vegetables Risotto" oncontextmenu="return false" onselectstart="return false" ondragstart="return false" onbeforecopy="return false" oncopy=document.selection.empty() onselect=document.selection.empty()>
-                    <div class="dish-info">
-                        <div class="name">Roasted Herb Vegetables Risotto</div>
-                        <div class="price">$58</div>
-                        <button class="add-to-cart-button">Add to Cart</button>
-                    </div>
-                </div>
-                <div class="dish">
-                    <img src="Sautéed Sesame Fish Pizza.jpg" alt="Sautéed Sesame Fish Pizza" oncontextmenu="return false" onselectstart="return false" ondragstart="return false" onbeforecopy="return false" oncopy=document.selection.empty() onselect=document.selection.empty()>
-                    <div class="dish-info">
-                        <div class="name">Sautéed Sesame Fish Pizza</div>
-                        <div class="price">$48</div>
-                        <button class="add-to-cart-button">Add to Cart</button>
-                    </div>
-                </div>
-                <div class="dish">
-                    <img src="Fried Herb Tofu Pizza.jpg" alt="Fried Herb Tofu Pizza" oncontextmenu="return false" onselectstart="return false" ondragstart="return false" onbeforecopy="return false" oncopy=document.selection.empty() onselect=document.selection.empty()>
-                    <div class="dish-info">
-                        <div class="name">Fried Herb Tofu Pizza</div>
-                        <div class="price">$28</div>
-                        <button class="add-to-cart-button">Add to Cart</button>
-                    </div>
+            <section class="dishes-display">
+                <div class="dishes-container" style="margin-left: 3%; margin-right: 3%;">
+                    <?php foreach ($dishes as $dish): ?>
+                        <div class="dish">
+                            <img src="<?php echo $dish['image'] ? $dish['image'] : 'nodish.jpg'; ?>" alt="<?php echo $dish['dish_name']; ?>" oncontextmenu="return false" onselectstart="return false" ondragstart="return false" onbeforecopy="return false" oncopy="document.selection.empty()" onselect="document.selection.empty()">
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Missing quotes in HTML event handlers</b></div>
<div id="fix">

Consider using HTML attributes with proper quotes. The `oncopy` and `onselect` event handlers are missing proper quotes which could cause issues in some browsers.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```
 @@ -226 +226 @@
 -                            <img src="<?php echo $dish['image'] ? $dish['image'] : 'nodish.jpg'; ?>" alt="<?php echo $dish['dish_name']; ?>" oncontextmenu="return false" onselectstart="return false" ondragstart="return false" onbeforecopy="return false" oncopy=document.selection.empty() onselect=document.selection.empty()>
 +                            <img src="<?php echo $dish['image'] ? $dish['image'] : 'nodish.jpg'; ?>" alt="<?php echo $dish['dish_name']; ?>" oncontextmenu="return false" onselectstart="return false" ondragstart="return false" onbeforecopy="return false" oncopy="document.selection.empty()" onselect="document.selection.empty()">
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Lux-Jason/rms_new/pull/6#issuecomment-2543172057>#ef655a</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged